src/bots/depositor_bot.py

```diff
@@ -230,8 +238,10 @@ def _buffered_ether_issue(self) -> bool:
         pending_gas_fee = self.w3.eth.get_block('pending').baseFeePerGas
         logger.info({'msg': 'Get pending `baseFeePerGas`.', 'value': pending_gas_fee})
 
-        buffered_ether = contracts.lido.functions.getBufferedEther().call(block_identifier=self.current_block.hash.hex())
-        logger.info({'msg': 'Call `getBufferedEther()`.', 'value': buffered_ether})
+        buffered_ether = contracts.lido.functions.getDepositableEther().call(
```

Comment:
Maybe rename variable to 'depositable_ether' ?

Reply:
Yeah, after upgrade we have to rewrite it because current check logic works good with buffered ether, but not with depositable ether. It will be changed further